packages/apps/frontera/src/store/Organizations/Organizations.store.ts

```diff
@@ -272,6 +272,10 @@ export class OrganizationsStore extends Store<OrganizationDatum, Organization> {
     try {
       const record = new Organization(this, Organization.default(payload));
 
+      record.value.enrichDetails = {
+        ...record.value.enrichDetails,
+        requestedAt: new Date().toISOString(),
+      };
```

Comment:
I would not do this here. Enriching status must be triggerd by backend invalidation not put by us automatically.